config/strategies/vkontakte.js

```diff
@@ -0,0 +1,24 @@
+var passport = require('passport'),
+    VkontakteStrategy = require('passport-vkontakte').Strategy,
+    config = require('../config'),
+    users = require('../../server-app-folder/controllers/users.server.controller');
+
```

Comment:
Better to add variables in this way.
let variable = val;
let variable = val;
...
let variable = val;